http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala

```diff
@@ -298,3 +300,62 @@ object GetRuntimeResponse {
     diskConfig
   )
 }
+
+final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRuntimeConfig],
+                                  appType: AppType,
+                                  diskConfig: Option[DiskConfigRequest],
+                                  labels: LabelMap = Map.empty)
+
+final case class GetAppResponse(kubernetesRuntimeConfig: KubernetesRuntimeConfig,
+                                errors: List[KubernetesError],
+                                status: AppStatus, //TODO: do we need some sort of aggregate status?
+                                proxyUrls: Map[ServiceName, URL],
```

Comment:
Yeah, so since we need to support multiple services, we will need multiple proxy urls. For example, to support routing to both cloudman and galaxy for an app. At first, there should only be one, but I want the UI to get used to looking it up in a map by a static service name